src/Lesson2.java

```diff
@@ -107,7 +105,7 @@ public static int isSumNegativeElements(int[] arrays) {
         return sumNegativeElements;
     }
 
-    public static int[] isDeleteArrayElement(int[] arrays, int deletePosition) {
+    public static int[] deleteArrayElement(int[] arrays, int deletePosition) {
```

Comment:
i is always < arrays.length so the condition is duplicates here
and what would happen if i pass deletePosition for example 50 and your array's length is 10???